config/daqconf_full_config.json

```diff
@@ -235,6 +235,8 @@
                 "time_before": 1000
             }
         },
+        "mlt_trigger_bitwords": [],
+        "mlt_use_bitwords": false,
```

Comment:
I'm wondering if it would be possible to make it so the `mlt_use_bitwords` is implied to be false anyway if the `mlt_trigger_bitwords` is empty, and true if not empty, so we wouldn't need it altogether (so it would be implied inside of the `get_trigger_app function`).
At the same time maybe it's better to have an explicit flag anyway? Meh.

Reply:
As discussed, keeping the flat gives more flexibility, such as testing with the flag on but no bitwords - ie no trigger decisions. Keeping as is for now.